lib/components/template/layer/LayerManager.vue

```diff
@@ -120,6 +125,14 @@ v-layout(column)
         this.layerToDelete = layer
       },
 
+      setLayerToggling(layer) {
+        this.toggleLayer(layer)
+      },
+
+      toggledIcon(layer) {
```

Comment:
I think this method is leftover and no longer used?

Reply:
Yeps, removed!